src/hooks/useDatasetManager.js

```diff
@@ -27,7 +27,7 @@ export const useDatasetManager = () => {
     token
   } = useContext(DatasetManagerContext)
   const { createToken, resetToken, validateToken } = useToken()
-  const [error, setError] = useState(false)
+  const [error, setError] = useState(null) // stores an error status code, otherwise null
```

Comment:
I don't think the comment in necessary here.

Reply:
Completed at 009323f324e84e42fc5cc47670ea1b3d32e92db3.